test/infrastructure/docker/exp/internal/controllers/dockermachinepool_controller.go

```diff
@@ -190,7 +195,7 @@ func (r *DockerMachinePoolReconciler) SetupWithManager(ctx context.Context, mgr
 			&clusterv1.Cluster{},
 			handler.EnqueueRequestsFromMapFunc(clusterToDockerMachinePools),
 			builder.WithPredicates(
-				predicates.ClusterUnpausedAndInfrastructureReady(mgr.GetScheme(), predicateLog),
+				predicates.ClusterInfrastructureReady(mgr.GetScheme(), predicateLog),
```

Comment:
Let's discuss this one as well. Here I lean towards simply just dropping builder.WithPredicates as it's just CAPD